openbb_platform/obbject_extensions/charting/openbb_charting/__init__.py

```diff
@@ -448,24 +437,12 @@ def to_chart(
         kwargs["symbol"] = symbol
         kwargs["target"] = target
         kwargs["index"] = index
-        kwargs["obbject_item"] = self._obbject.results
-        kwargs["charting_settings"] = self._charting_settings
-        if (
-            hasattr(self._obbject, "_standard_params")
-            and self._obbject._standard_params  # pylint: disable=protected-access
-        ):
-            kwargs["standard_params"] = (
-                self._obbject._standard_params.__dict__  # pylint: disable=protected-access
-            )
-        kwargs["provider"] = self._obbject.provider  # pylint: disable=protected-access
-        kwargs["extra"] = self._obbject.extra  # pylint: disable=protected-access
-        metadata = kwargs["extra"].get("metadata")
-        kwargs["extra_params"] = (
-            metadata.arguments.get("extra_params") if metadata else None
-        )
-        if "kwargs" in kwargs:
-            _kwargs = kwargs.pop("kwargs")
-            kwargs.update(_kwargs.get("chart_params", {}))
+        kwargs["obbject_item"] = getattr(self._obbject, "results", [])
+        kwargs["charting_settings"] = getattr(self, "_charting_settings", {})
+        kwargs["standard_params"] = getattr(self._obbject, "_standard_params", {})
+        kwargs["extra_params"] = getattr(self._obbject, "extra_params", {})
+        kwargs["provider"] = getattr(self._obbject, "provider", "")
```

Comment:
We can simply access the attributes without using `getattr`, since we know the `OBBject` have them.